src/components/shared/formItems/EthAddress.tsx

```diff
@@ -74,16 +74,28 @@ export default function EthAddress({
 
     readENSName()
   }, [defaultValue])
-
+  console.log('formItemProps?.extra: ', formItemProps?.extra)
   return (
-    <Form.Item {...formItemProps}>
+    <Form.Item
+      {...formItemProps}
+      className={
+        formItemProps?.extra
+          ? 'ant-form-item-control-input-extra-and-error'
```

Comment:
i may be missing something with your CSS solution here, but should we just override this for all cases? as in, could we apply the CSS for `.ant-form-item-control-input-extra-and-error` to all form item elements?

Reply:
@peripheralist oddly the FormItem places the error message in the same spot regardless of whether an 'extra' is present or not. So in cases where no 'extra' is present (such as 'Add a payout'), and we give it the class '.ant-form-item-control-input-extra-and-error', the error message appears far too low (see attached screenshot). So different CSS is required when we choose not to use the 'extra' prop. 

'Add a payout' (no extra) with the same CSS as other cases with an extra 
<img width="527" alt="Screen Shot 2021-12-22 at 6 05 51 pm" src="https://user-images.githubusercontent.com/96150256/147057624-da1fd2d8-13ef-4091-a14b-6b27424d7b38.png">

Reply:
got it, thanks for clarifying. add this to the list of Antd funniness we should try to build around (cc @torvusbug)